SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialActorChannel.cpp

```diff
@@ -1019,7 +1019,6 @@ void USpatialActorChannel::PostReceiveSpatialUpdate(UObject* TargetObject, const
 													const TMap<GDK_PROPERTY(Property) *, FSpatialGDKSpanId>& PropertySpanIds)
 {
 	FObjectReplicator& Replicator = FindOrCreateReplicator(TargetObject).Get();
-	TargetObject->PostNetReceive();
```

Comment:
splitting out the postnetreceive from postreceivespatialupdate as postnetreceives for different actors can't overlap (AActor::PostNetReceive does some stuff with static variables).

I might bring them back together with a later PR but will be a big change to actorsystem

Reply:
can we find out who wrote/owns this code by looking at git and invite them here to check the workings?
I expect it will be Mike or Valentyn, or this will be ancient code.

Reply:
seems to have been @improbable-valentyn (I know you're on gearbox tho valy!)

Reply:
I think it was @improbable-valentyn (though know you're not working on gdk atm!)